index.js

```diff
@@ -4,7 +4,45 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let arr = process.argv.slice(2);
+    let sum = Number(arr[1]);
+
+    if (arr.length === 0) {
+        return "No operation provided...";
+    } else if (arr[0] !== 'minus' && arr[0] !== 'plus') {
+        return "Invaild operation: ${arr[0]}";
```

Comment:
It looks like you tried to use a template literal here but used the wrong syntax. With template literals you want to use back ticks instead of the double quotations marks. 

Also, you misspelled invalid which will cause the test to fail.